optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingDomainModel/CloudBalancingDomainModel-section.adoc

```diff
@@ -8,25 +8,30 @@
 
 Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. It also helps to simplify contraints, improve performance, and increase flexibility for future needs. 
 
+To create a domain model, define all the objects that represent the input data for the problem. In this simple example, the objects are processes and computers. 
+
+A separate object in the domain model must represent a full data set of problem, which contains the input data as well as a solution. In this example, this object holds a list of computers and a list of processes. Each process is assigned to a compter; the distribution of processes between computers is the solution.
+
 . Draw a class diagram of your domain model. 
 . Normalize it to remove duplicate data.
-. Write down some sample instances for each class. 
+. Write down some _sample instances_ for each class. Sample instances are entity properties that are relevant for planning purposes.
```

Comment:
"Same instances are entity properties" is terminologically incorrect.

A property is a getter+setter (or getter alone) of Java Bean. Instances are class instances. Clearly differentiate these to use them correctly:
- class, for example Person.java
- (class) instance, for example Ann
- getter, for example Person.getAge()
- setter, for example Person.setAge()
- property, for example getAge()/setAge() on Person
- field, for example Person.age
- value, for example 37 (Ann's age)

Reply:
In this case could you clarify what a "sample instance" actually is? Is a "sample instance" simply an example of a class with all its properties set? I'll get that into the text.

Reply:
What got me confused, perhaps, is this part:

''''
In this example, the sample instances for the `Computer` class are: `cpuPower`, `memory`, `networkBandwidth`, `cost`.

Sample instances for ``Process`` are: `requiredCpuPower`, `requiredMemory`, and `requiredNetworkBandwidth`.
''''

These are properties, aren't they? Then why are they called sample instances?

Reply:
I can't leave it unchanged because it is unclear. It now says that sample instances for the Computer class are cpuPower etc. However your later definition of instances does not work. What it describes as "sample instances" are not instances.

I understand my version is wrong but I need to work out the right version.

Reply:
As requested in email, rolled back for this PR but will submit next version in a separate PR.